packages/python/plotly/plotly/figure_factory/_annotated_heatmap.py

```diff
@@ -147,9 +147,10 @@ def create_annotated_heatmap(
 
 
 def to_rgb_color_list(color_str, default):
-    if "rgb" in color_str:
-        return [int(v) for v in color_str.strip("rgb()").split(",")]
-    elif "#" in color_str:
+    color_str = color_str.strip()
+    if color_str.startswith("rgb"):
+        return [int(v) for v in color_str.strip("rgba()").split(",")]
```

Comment:
does this now still work with `rbg(...)` and `rgba(...)` both ?

Reply:
Yes, it does. Tested it. As I mentioned, doesn't handle HSL(A) yet.

Reply:
OK. After this next release I won't be accepting any more PRs to this deprecated code so let's not work on any HSL stuff please :)